ThePensionsRegulator.Frontend/Views/Shared/TPR/TPRHeaderLockup.cshtml

```diff
@@ -29,6 +30,36 @@
                 {
                     <tpr-header-search action="@Model.HeaderSearchActionPath" autocomplete-url="@Model.HeaderSearchAutocompleteUrl" placeholder="@Model.HeaderSearchPlaceholderText" aria-label="@Model.HeaderSearchAriaLabel" input-name="@Model.HeaderSearchInputName"></tpr-header-search>
                 }
+                @if (Model.ShowHeaderMenu)
```

Comment:
I think the menu option is meant to be mutually exclusive with the `HeaderBarContent` option (which normally contains some links), so you should add a check for that to be null, similar to the one shown for the search option a few lines earlier.